js/runner.go

```diff
@@ -347,9 +347,10 @@ func (r *Runner) GetOptions() lib.Options {
 
 // IsExecutable returns whether the given name is an exported and
 // executable function in the script.
+//
+// TODO: completely remove this?
```

Comment:
Actually, I am not sure if we can remove this without a lot more refactoring :thinking: `Bundle.IsExecutable` is not available everywhere as a substitute... :thinking:

Reply:
I dug a little more here and realized that the need for the public method in `Bundle` disappears because of the other simplifications I did in https://github.com/grafana/k6/pull/2412 with `loadedTest` :tada: So I ended up reverting this part of the change there, see https://github.com/grafana/k6/pull/2412/commits/c3fec322bb00515b7c4643365a3e2f274414bdba

I think it's fine to merge this as it is and remove it in the other PR, given that it allows us to merge these PRs one by one instead of all at once.